exercises/hello-world/hello_world.cpp

```diff
@@ -0,0 +1,16 @@
+#include "hello_world.h"
+
+// Use everything from the 'std' namespace.
+// This lets us write 'string' instead of 'std::string'.
+using namespace std;
```

Comment:
Is that really a good idea? This is actually suggesting to use `using namespace std;` all the time. Not a good coding style

Reply:
People tend to disagree on this point - See [CppCoreGuidelines](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#sf6-use-using-namespace-directives-for-transition-for-foundation-libraries-such-as-std-or-within-a-local-scope-only) and [Google](https://abseil.io/tips/153).

It's generally safe to use in a small, self-contained source file, though there are potential issues in large codebases. It is one of those conveniences that I see as being very helpful while learning the language and potentially dangerous later. You're going to almost never hit issues from it in a toy problem.

That said, I mostly use this in toy problems because I'm lazy and it's low- to no-risk. Do you think it would be better to encourage new learners to use `using std::string;` instead of the whole namespace?